src/radical/pilot/utils/analysis.py

```diff
@@ -569,6 +570,9 @@ def get_info_df(df):
         for i in set(l):
             cols.add(i)
             if l.count(i)>1:
+                # TODO: this is a hack to workaround a problem in the state management
+                if i == 'mod_adv_u':
```

Comment:
I noticed. Although even if it gets fixed, we probably want to keep it around for a bit for existing profiles.